internal/services/postgres/postgresql_server_resource.go

```diff
@@ -204,7 +210,8 @@ func resourcePostgreSQLServer() *pluginsdk.Resource {
 			"auto_grow_enabled": {
 				Type:          pluginsdk.TypeBool,
 				Optional:      true,
-				Computed:      true, // TODO: remove in 3.0 and default to true
+				Default:       features.ThreePointOhBetaResources(),
+				Computed:      !features.ThreePointOhBetaResources(),
```

Comment:
pretty sure a field can't be both Defaulted and Computed, else the Plugin SDK complains on launch, so we may need to conditionally output different schemas here:

```suggestion
			"auto_grow_enabled": func() *schema.Schema {
				if !features.ThreePointOh() {
					return &schema.Schema{
 						Type:          pluginsdk.TypeBool,
 						Optional:      true,
						Default:       true,
						ConflictsWith: []string{"storage_profile", "storage_profile.0.auto_grow"},
					}
				}
				
				return &schema.Schema{
 					Type:          pluginsdk.TypeBool,
 					Optional:      true,
					Computed:      true,
					ConflictsWith: []string{"storage_profile", "storage_profile.0.auto_grow"},
				}
			}(),
```